EIP-draft.md

```diff
@@ -34,30 +34,29 @@ These curves could represent:
 
 ### Use Cases
 
-1. A framework  to release tokens over a set period of time that can be used to build many kinds of NFT financial products such as bonds, treasury bills, and many others.
+1. A framework to release tokens over a set period of time that can be used to build many kinds of NFT financial products such as bonds, treasury bills, and many others.
 2. Replicating SAFT contracts in a standardized form of semi-liquid vesting NFT assets.
-    - SAFTs are generally off-chain, while today's on-chain versions are mainly address-based, which makes distributing vesting shares to many representatives difficult. Standardization simplifies this convoluted process.
+   - SAFTs are generally off-chain, while today's on-chain versions are mainly address-based, which makes distributing vesting shares to many representatives difficult. Standardization simplifies this convoluted process.
 3. Providing a path for the standardization of vesting and token timelock contracts.
-    - There are many such contracts in the wild and most of them differ in both interface and implementation.
+   - There are many such contracts in the wild and most of them differ in both interface and implementation.
 4. NFT marketplaces dedicated to vesting NFTs.
-    - Whole new sets of interfaces and analytics could be created from a common standard for token vesting NFTs.
+   - Whole new sets of interfaces and analytics could be created from a common standard for token vesting NFTs.
 5. Integrating vesting NFTs into services like Gnosis Safe.
-    - A standard would mean services like Gnosis Safe could more easily and uniformly support interactions with these types of contracts inside of a multisig contract.
+   - A standard would mean services like Gnosis Safe could more easily and uniformly support interactions with these types of contracts inside of a multisig contract.
 6. Enable standardized fundraising implementations and general fundraising that sell vesting tokens (eg. SAFTs) in a more transparent manner.
 7. Allows tools, front-end apps, aggregators, etc. to show a more holistic view of the vesting tokens and the properties available to users.
-    - Currently, every project needs to write their own visualization of the vesting schedule of their vesting assets. If this is standardized, third-party tools could be developed aggregate all vesting NFTs from all projects for the user, display their schedules and allow the user to take aggregated vesting actions.
-    - Such tooling can easily discover compliance through the [EIP-165](./eip-165.md) `supportsInterface(InterfaceID)` check.
+   - Currently, every project needs to write their own visualization of the vesting schedule of their vesting assets. If this is standardized, third-party tools could be developed aggregate all vesting NFTs from all projects for the user, display their schedules and allow the user to take aggregated vesting actions.
```

Comment:
I also noticed that "third-party tools could be developed aggregate" should be: developed "to" aggregate